tests/SimplesheetServiceProviderTest.php

```diff
@@ -4,7 +4,7 @@
 
 use Nikazooz\Simplesheet\Simplesheet;
 
-class SpreadsheetServiceProviderTest extends TestCase
+class SimplesheetServiceProviderTest extends TestCase
```

Comment:
This is now in master. Couldn't wait for everything in this PR to be ready, but still thank you